Project/frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine 
+
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
Czy jest jakiś konkretny powód na kopiowanie locka, gdy będziemy instalować od zera? Jeśli tak, to czemu nie jest to w jednej komendzie? W taki sposób tworzymy niepotrzebnie dodatkowy krok, który jeszcze zajmuje pamięć, bo jest cacheowany :smile: